.vitepress/theme/style/index.scss

```diff
@@ -9,7 +9,7 @@ $iroha-brand-dark: rgba(187, 2, 24, 1);
 }
 
 :root {
-  --vp-font-family-mono: 'JetBrains Mono', 'FiraCode', Menlo, Monaco, Consolas, 'Courier New', monospace;
+  --vp-font-family-mono: 'JetBrains Mono', 'Fira Code', Menlo, Monaco, Consolas, 'Courier New', monospace;
```

Comment:
I'd recommend avoiding `Fira Code` because of the ligatures. Using them in your personal projects/IDE is fine. Using them in a tutorial is not.

Reply:
Disabled. Btw, `JetBrains Mono` also uses ligatures.